jaba/src/App.tsx

```diff
@@ -1,9 +1,11 @@
 import ExamplePage from "./pages/ExamplePage/ExamplePage";
+import ForgotPassword from "./pages/ForgotPasswordFrontEnd";
```

Comment:
You put your files in subdirectories under pages/ like I asked but you also need to update the import paths in the App.tsx file so errors don't occur on build